spec/common.ts

```diff
@@ -1,110 +1,110 @@
-var webdriver = require('selenium-webdriver');
+import {promise as wdpromise, WebElement} from 'selenium-webdriver';
```

Comment:
It was important back when I was using `noImplicitAny`, and will be important again in https://github.com/angular/jasminewd/pull/79.  Removed from this PR and transferred over